apps/condo/domains/news/components/NewsForm/InputStep/InputStepSelector.tsx

```diff
@@ -0,0 +1,329 @@
+/** @jsx jsx */
+import { useGetNewsSharingRecipientsLazyQuery } from '@app/condo/gql'
+import { B2BAppNewsSharingConfig, Property as IProperty,  NewsItem as INewsItem } from '@app/condo/schema'
+import { css, jsx } from '@emotion/react'
+import { Col, Form, FormInstance, notification, Row } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import get from 'lodash/get'
+import has from 'lodash/has'
+import isEmpty from 'lodash/isEmpty'
+import React, { useCallback, useMemo, useRef } from 'react'
+
+import { getClientSideSenderInfo } from '@open-condo/codegen/utils/userId'
+import { useIntl } from '@open-condo/next/intl'
+import { Alert, Typography } from '@open-condo/ui'
+
+import { GraphQlSearchInput } from '@condo/domains/common/components/GraphQlSearchInput'
+import {
+    GraphQlSearchInputWithCheckAll,
+    InputWithCheckAllProps,
+} from '@condo/domains/common/components/GraphQlSearchInputWithCheckAll'
+import { LabelWithInfo } from '@condo/domains/common/components/LabelWithInfo'
+import { useValidations } from '@condo/domains/common/hooks/useValidations'
+import { HiddenBlock, ScopeType } from '@condo/domains/news/components/NewsForm/BaseNewsForm'
+import { SectionNameInput } from '@condo/domains/user/components/SectionNameInput'
+import { UnitNameInput, UnitNameInputOption } from '@condo/domains/user/components/UnitNameInput'
+
+const SMALL_VERTICAL_GUTTER: [Gutter, Gutter] = [0, 24]
+const FORM_FILED_COL_PROPS = css`width: 100%; padding: 0; height: 44px`
```

Comment:
move to `css` file

Reply:
Do we really need this?
What are these styles for?
Can we fix this directly in the "GraphQlSearchInputWithCheckAll" component?